packages/http/src/__tests__/http-prism-instance.spec.ts

```diff
@@ -66,8 +66,8 @@ describe('Http Client .request', () => {
     });
 
     describe('baseUrl not set', () => {
-      it('ignores server validation and returns 200', () => {
-        assertResolvesRight(
+      it('ignores server validation and returns 200', async () => {
+        await assertResolvesRight(
```

Comment:
@XVincentX I added those because rejections from this test were caught inside a test which is quite far from here. I guess we should be strict about promises and handle them properly inside tests.

Reply:
I had mixed feeling about whether to use `return` or `await`.
If I use `return` then in case of adding an expect it is very likely for someone to forget that the code is actually asynchronous. If I use await I'm clearly static that what we are doing here is `async`. But it was not a strong opinion, I'll change it to `return`.